pinot-core/src/main/java/com/linkedin/pinot/core/segment/index/loader/SegmentPreProcessor.java

```diff
@@ -62,6 +63,16 @@ public void process() throws Exception {
       return;
     }
 
+    File[] directoryListing = _indexDir.listFiles();
+    String tempFileExtension = V1Constants.Indexes.BITMAP_INVERTED_INDEX_FILE_EXTENSION + ".tmp";
+    if (directoryListing != null) {
+      for (File child : directoryListing) {
+        if (child.getName().endsWith(tempFileExtension)) {
+          org.apache.commons.io.FileUtils.deleteQuietly(child);
```

Comment:
You can import this class instead of putting the full path (seems no class name collision)